content/influxdb/v2/get-started/setup.md

```diff
@@ -310,16 +456,26 @@ Set the following environment variables in your command line session:
 - `INFLUX_ORG_ID`: InfluxDB [organization ID](/influxdb/v2/admin/organizations/view-orgs/#view-your-organization-id).
 - `INFLUX_TOKEN`: InfluxDB API token.
 
+{{% code-placeholders "&lt;(ORG_NAME|ORG_ID|API_TOKEN)&gt;" %}}
+
 ```sh
 export INFLUX_HOST=http://localhost:8086
```

Comment:
```suggestion
export INFLUX_HOST={{< influxdb/host >}}
```

Reply:
> `+ export INFLUX_HOST={{< influxdb/host >}}`

The shortcode has an error in the CI build:

```
failed to process shortcode: "/home/circleci/project/layouts/shortcodes/influxdb/host.html:8:20": execute of template failed: template: shortcodes/influxdb/host.html:8:20: executing "shortcodes/influxdb/host.html" at <index .Site.Data.products $productRef>: error calling index: index of type map[string]interface {} with args [<nil>] failed: value is nil; should be of type string
```